rtengine/imagefloat.h

```diff
@@ -214,7 +214,7 @@ class Imagefloat final : public IImagefloat, public ImageIO
         exponent += (128 - 64);
         mantissa <<= 7;
         // Assemble sign, exponent and mantissa.
-        return (uint32_t) ((sign << 31) | (exponent << 23) | mantissa);
+        return (uint32_t) ((lsign << 31) | (exponent << 23) | mantissa);
```

Comment:
Does this really have to be inline in the (or *this*) header?

Reply:
As above